connectors/webhook/src/main/java/io/camunda/connector/inbound/HttpWebhookExecutable.java

```diff
@@ -43,6 +46,43 @@
 import org.slf4j.LoggerFactory;
 
 @InboundConnector(name = "Webhook", type = "io.camunda:webhook:1")
+@ElementTemplate(
+    id = "io.camunda.connectors.webhook",
+    name = "Webhook Connector",
+    icon = "icon.svg",
+    version = 10,
+    inputDataClass = WebhookConnectorPropertiesWrapper.class,
+    description = "Configure webhook to receive callbacks",
+    documentationRef =
+        "https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/http-webhook/",
+    propertyGroups = {
+      @PropertyGroup(id = "endpoint", label = "Webhook configuration"),
+      @PropertyGroup(id = "authentication", label = "Authentication"),
+      @PropertyGroup(id = "authorization", label = "Authorization"),
+      @PropertyGroup(id = "webhookResponse", label = "Webhook response")
+    },
+    elementTypes = {
+      @ConnectorElementType(
+          appliesTo = BpmnType.START_EVENT,
+          elementType = BpmnType.START_EVENT,
+          templateIdOverride = "io.camunda.connectors.webhook.WebhookConnector.v1",
+          templateNameOverride = "Webhook Start Event Connector"),
```

Comment:
I am not 100% sure, but feels like we wanted to keep the first word and the `Connector` capitalized, in this case smth like: 

```suggestion
          templateNameOverride = "Webhook start event Connector"),
```

Maybe @YanaSegal can confirm or disproof?

Reply:
Looking at other connectors, we are largely using capital case so I'd rather leave it as is and tackle possible renaming separately if we agree on that

Reply:
Sure